### PR TITLE
Contact form feature

### PR DESCRIPTION
**Pull request by kamranasif506**
Following features have been added

- [ ] Contact Us form
- [ ] Footer

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,208 +1,274 @@
...
+            ><img src="images/ic_medium.png" alt="" class="flex-item"
+          /></a>
+        </div>
+        <div>
+          <a href="https://www.angellist.com/"
+            ><img src="images/ic_angellist.png" alt="" class="flex-item"
+          /></a>
+        </div>
+      </div>
+    </footer>
```
Comment (stanleySimeon) [inline index.html:272]: - [ ] Kindly I suggest you add a label for each input as shown in the Figma design.

- [ ] Kindly make sure the form is validated by using a correct `formspree` id. In your case I see the id is not complete, kindly fix that. 😄

Screenshot 2023-05-02 at 15 34 33

- [ ] Kindly make sure you stick to the design templates in Figma as much as possible, this includes the font, colors, images, and spacing (margin/padding). Please follow this [link](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23-9&t=LVMtdwmoeA5knQai-0) to see the Figma template.

> ### The Figma design
>
>Screenshot 2023-05-02 at 15 39 48
>
>### Your Design
>
>Screenshot 2023-05-02 at 15 40 34

**Review by stanleySimeon — CHANGES_REQUESTED** [review]
# Hi @kamranasif506, @Zafron047 
 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Correct use of Gitflow ✔️
- Linter checks are passed ✔️
- Good PR description ✔️
- Meaningful commit messages ✔️

# Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

Cheers, and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @stanleySimeon in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (kamranasif506) [reply index.html:272]: @stanleySimeon  Hello sir, i think there might be some mistake as this is not my design. can you please check it again.

**Review by iambenkis — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @iambenkis in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**